src/Resources/config/form.xml

```diff
@@ -5,16 +5,16 @@
             <tag name="form.type" alias="sonata_formatter_type"/>
             <argument type="service" id="sonata.formatter.pool"/>
             <argument type="service" id="translator"/>
-            <argument type="service" id="ivory_ck_editor.config_manager"/>
-            <argument type="service" id="ivory_ck_editor.plugin_manager"/>
-            <argument type="service" id="ivory_ck_editor.template_manager"/>
+            <argument type="service" id="fos_ck_editor.config_manager"/>
+            <argument type="service" id="fos_ck_editor.plugin_manager"/>
+            <argument type="service" id="fos_ck_editor.template_manager"/>
```

Comment:
Please make this dynamic somehow (either do it in php or load an xml file conditionally)

Reply:
Make 2 `form.xml` one for fos, one for ivory?